src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/HeadCracker.java

```diff
@@ -0,0 +1,69 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.items;
+
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectionModule;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.Setup.Messages;
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import org.bukkit.Material;
+import org.bukkit.Sound;
+import org.bukkit.SoundCategory;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Lists.SlimefunItems;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.interfaces.DamageableItem;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.interfaces.NotPlaceable;
+import me.mrCookieSlime.Slimefun.Objects.handlers.ItemInteractionHandler;
+import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+
+public class HeadCracker extends SimpleSlimefunItem<ItemInteractionHandler> implements NotPlaceable, DamageableItem {
+
+    private boolean damageOnUse;
+
+    public HeadCracker(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe, String[] keys, Object[] values) {
+        super(category, item, id, recipeType, recipe, keys, values);
+    }
+
+    @Override
+    public ItemInteractionHandler getItemHandler() {
+        return (e, p, item) -> {
+            if (SlimefunManager.isItemSimiliar(item, SlimefunItems.HEAD_CRACKER, true)) {
+                Block b = e.getClickedBlock();
+                if (SlimefunPlugin.getProtectionManager().hasPermission(p, b.getLocation(), ProtectionModule.Action.BREAK_BLOCK)) {
```

Comment:
You can just use Action.BREAK_BLOCK
No need to write ProtectionModule.Action.BREAK_BLOCK

Reply:
Action is a subclass of ProtectionModule so nope I must do Action.BREAK_BLOCK. Btw it's an auto change made by Intellij

Reply:
Again, you can do both. Both work.
But the first method is shorter and used throughout all of Slimefun4, so this one line doing it differently doesn't make much sense.
And I am sure you can turn that off in IntelliJ